.github/workflows/test.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - main
+      - 3670_use_pytest_importlib
```

Comment:
@valeriupredoi Could you clean this up please, so we can get this merged?

Reply:
on it!

Reply:
done now :beer:

Reply:
Thanks!